packages/get-platform/src/getPlatform.ts

```diff
@@ -156,34 +156,33 @@ type GetOpenSSLVersionParams = {
  * On Linux, returns the OpenSSL version excluding the patch version, e.g. "1.1.x".
  * Reading the version from the libssl.so file is more reliable than reading it from the openssl binary.
  * This function never throws.
- *
- * TODO: we should probably validate the output of this function, and ensure it's contained in `supportedLibSSLVersions`
  */
 export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetOSResult['libssl'] | undefined> {
-  const libsslSpecificPaths = await match(args)
-    .with({ distro: 'musl' }, async () => {
+  const archFromUname = await getArchFromUname()
+
+  const libsslSpecificPaths = match(args)
+    .with({ distro: 'musl' }, () => {
       /* Linux Alpine */
       debug('Trying platform-specific paths for "alpine"')
-      return Promise.resolve(['/lib'])
+      return ['/lib']
     })
-    .with({ distro: 'debian' }, async () => {
+    .with({ distro: 'debian' }, () => {
       /* Linux Debian, Ubuntu, etc */
-      const archFromUname = await getArchFromUname()
       debug('Trying platform-specific paths for "debian" (and "ubuntu")')
       return [`/usr/lib/${archFromUname}-linux-gnu`, `/lib/${archFromUname}-linux-gnu`]
     })
-    .with({ distro: 'rhel' }, async () => {
+    .with({ distro: 'rhel' }, () => {
       /* Linux Red Hat, OpenSuse etc */
       debug('Trying platform-specific paths for "rhel"')
-      return Promise.resolve(['/lib64', '/usr/lib64'])
+      return ['/lib64', '/usr/lib64']
     })
     .otherwise(({ distro, arch }) => {
       /* Other Linux distros, we don't do anything specific and fall back to the next blocks */
       debug(`Don't know any platform-specific paths for "${distro}" on ${arch}`)
-      return Promise.resolve(undefined)
+      return undefined
```

Comment:
Would it be better to return an empty array here and remove the default value below on line 185?

Reply:
Yes, we can do that